runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -99,7 +101,7 @@ public void resetCaches() {
      *
      * @param address
      */
-    private void validateStateOfWrittenEntry(long address) {
+    private void validateStateOfWrittenEntry(long address, Throwable t) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "t". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)